pom.xml

```diff
@@ -854,12 +859,13 @@
             <properties>
                 <envClassifier>hdp3.1.0</envClassifier>
                 <envHbase>hbase2.0.2</envHbase>
+                <envSpark>spark2.3</envSpark>
```

Comment:
We already have _spark.version_ and _spark.major.version_. Why do we need yet another one, envSpark (matching spark.major.version)?

Reply:
Good point. I did not noticed spark.major.version is defined